### PR TITLE
Use & enforce buildifier + fix all formatting and linting warnings

### LINKED ISSUE #11: Enforce formatting with Buildifier
I noticed [`lib.star`](https://github.com/cirrus-modules/helpers/blob/main/lib.star) is not formatted with [Buildifier](https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md). It's like `gofmt` and `govet` but for Starlark.

### PR DESCRIPTION
**Pull request by bartekpacia**
This PR resolves #11

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -2,4 +2,10 @@ task:
   name: Test
   container:
     image: ghcr.io/cirruslabs/cirrus-cli:latest
-  script: cirrus internal test
+  test_script: cirrus internal test
+  setup_buildifier_script: |
+    apk add go
+    go install github.com/bazelbuild/buildtools/buildifier@latest
```
Comment (fkorotkov) [inline .cirrus.yml:8]: @edigaryev I wonder if we should just include it into Cirrus CLI and add commands like `cirrus internal format` and `cirrus internal format --check` to check in a standard way. What do you think?

Comment (fkorotkov) [reply .cirrus.yml:8]: Or `cirrus internal lint` instead of `cirrus internal format --check`

Comment (bartekpacia) [reply .cirrus.yml:8]: I'm not @edigaryev but I think that's a great idea! :D

Comment (edigaryev) [reply .cirrus.yml:8]: I propose to keep it this way for now, until there's a wider demand for such a command in the Cirrus CLI.

The [`buildifier.go` command](https://github.com/bazelbuild/buildtools/blob/master/buildifier/buildifier.go) implements quite a lot of logic (on top of the [publicly exposed package](https://pkg.go.dev/github.com/bazelbuild/buildtools/buildifier)) which we'd need to copy/re-implement in the Cirrus CLI.

**Review by edigaryev — APPROVED** [review]

**Pull request merged**